packages/hub/services/scheduled-payments/fetcher.ts

```diff
@@ -5,7 +5,7 @@ import { convertDateToUTC } from '../../utils/dates';
 
 export default class ScheduledPaymentsFetcherService {
   prismaManager = inject('prisma-manager', { as: 'prismaManager' });
-  validForDays = 10; // TODO: Keep this in sync with the value in the config contract
+  validForDays = 3;
```

Comment:
We can read this from the config contract. But if we don't anticipate this will change we can also leave it hardcoded for now.